docs/toolbox/gpu_operator.rst

```diff
@@ -139,12 +139,12 @@ in gpu-operator-resources, ...)
 
 .. code-block:: shell
 
-    toolbox/entitlement/test_cluster.sh
-    toolbox/nfd/has_nfd_labels.sh
-    toolbox/nfd/has_gpu_nodes.sh
-    toolbox/gpu-operator/wait_deployment.sh
-    toolbox/gpu-operator/run_gpu_burn.sh 30
-    toolbox/gpu-operator/capture_deployment_state.sh
+    ./run_toolbox.py entitlement test_cluster
+    ./run_toolbox.py nfd has_labels
+    ./run_toolbox.py nfd has_gpu_nodes
+    ./run_toolbox.py gpu_operator wait_deployment
+    ./run_toolbox.py gpu_operator run_gpu_burn 30
```

Comment:
I think you need `--runtime=30` here

Reply:
> hey @omertuc , seems pretty good to me!
> maybe we can merge it before you fix the two docs typo I mentioned above, as all the tests are green?
> 
> \lgtm
> \hold # I let you merge if you're good with everything you had in mind for this PR

`s@\@/g`

Reply:
It's not needed, but it's nice if you want to be more explicit